### PR TITLE
[WIP] Fixed netlayer and tor binary dependency issues

### PR DESCRIPTION
**Pull request by freimair**
netlayer had and has a well-hidden build configuration issue since version 0.4.2. It always fetches the "master" binaries instead of the binaries of the respective release. although until recently, bisq wanted to use 0.4.2, but actually has used 0.4.7 already. Due to changes in netlayer and its upcoming release, the netlayer build configuration has changed and thus, the issue showed itself.

This is an attempt to fix the issue temporarily. I, however, do expect failures due to checksum mismatches. Work in progress.

If this works, I will amend to the request and make things more final, i.e. create a proper release of netlayer and include that.

### CODE REVIEW

**Comment by freimair** [comment]
please do not merge this pull-request. I will supply a clean one if everything works again.

just using the travis CI here

**Comment by ManfredKarrer** [comment]
Thanks for looking into it!

File: build.gradle [hunk 1]
```diff
@@ -37,9 +37,9 @@ configure(subprojects) {
     }
 
     repositories {
-        mavenCentral()
```
Comment (cbeams) [inline build.gradle]: Please don't change this. There are issues with JCenter's SSL cert under OpenJDK 10, that's why I recently switched everything to Maven Central.

File: build.gradle [hunk 2]
```diff
@@ -37,9 +37,9 @@ configure(subprojects) {
...
-        maven { url 'https://jitpack.io' }
-        maven { url 'https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/' }
```
Comment (cbeams) [inline build.gradle:42]: Please don't kill the whitespace after the opening brace.

**Review by cbeams — CHANGES_REQUESTED** [review]
NACK per comments. Minor stuff, but the Maven Central change is more than just a nit and should actually be changed. Also, please review rules 3 and 5 from the commit comment guidelines at https://github.com/bisq-network/style/issues/9 (no need to rebase/reword the commits in this PR, just for future reference). Thanks.

**Comment by cbeams** [comment]
Sorry, @freimair, I just noticed this is still WIP and that you said you'd clean it up once you know everything is working.

**Comment by ManfredKarrer** [comment]
@freimair @cbeams Let me know when its ready to merge/ACKed by @cbeams (or @cbeams  - merge yourself).

**Comment by freimair** [comment]
working again. I am going to close this mess and open a clean one.

**Pull request closed without merging**